app-route.html

```diff
@@ -399,7 +399,14 @@
 
         for (var property in setObj) {
           this._pathEffector(property, this[property]);
-          this._notifyPathUp(property, this[property]);
+          var rootName = this._modelForPath(property);
```

Comment:
If that's really what you want, probably just use `_notifyChange` from https://github.com/Polymer/polymer/blob/master/src/lib/bind/accessors.html#L21  E.g.

```
   this._notifyChange(property);
```
(Is this `property` a property name as the variable name suggest or really a path? B/c `property === this._modelForPath(property)`) If it's a property `_effectEffects` instead of `_pathEffector` should be used (which in turn shoould also automatically call `_notifyChange` if notifying upwards is indeed a registered effect on that property. Otherwise if it's a path using `_propertySetter` is (as the name of the method and it's origin in the Polymer source code suggest) misleading.